packages/grid/x-data-grid/src/components/panel/GridPreferencesPanel.tsx

```diff
@@ -28,12 +24,10 @@ export const GridPreferencesPanel = React.forwardRef<
       {...props}
       {...rootProps.componentsProps?.basePopper}
     >
-      {!rootProps.disableColumnSelector && isColumnsTabOpen && (
-        <rootProps.components.ColumnsPanel {...rootProps.componentsProps?.columnsPanel} />
-      )}
-
-      {!rootProps.disableColumnFilter && isFiltersTabOpen && (
-        <rootProps.components.FilterPanel {...rootProps.componentsProps?.filterPanel} />
+      {apiRef.current.unstable_applyPreProcessors(
+        'preferencePanel',
+        null,
+        preferencePanelState.openedPanelValue ?? GridPreferencePanelsValue.filters,
       )}
     </rootProps.components.Panel>
```

Comment:
That's a bit weird to see `apiRef.current.unstable_applyPreProcessors` called inside the rendering

Could be more readable as follow, but not 100% sure it's the same
```jsx
const panelContent = apiRef.current.unstable_applyPreProcessors(...);
...
<rootProps.components.Panel ...>
	{panelContent}
</rootProps.components.Panel>
```

Reply:
It is the same yes
Done